### PR TITLE
docs(code): update comments of dispatch property of EffectOptions

### PR DESCRIPTION
**Pull request by juri-sinitson**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/datorama/akita/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #612 

## What is the new behavior?
Updating comments of dispatch property of EffectOptions

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by juri-sinitson** [comment]
Running tests (`yarn test:all`) and building libs (``yarn build:libs`) is successful, building apps (`yarn build:apps`) fails wth:
```
ERROR: libs/akita/src/lib/store.ts:48:14 - error NG2007: Class is using Angular features but is not decorated. Please add an explicit Angular decorator.

48 export class Store<S = any> {
                ~~~~~

libs/akita/src/lib/store.ts:48:14 - error NG2007: Class is using Angular features but is not decorated. Please add an explicit Angular decorator.

48 export class Store<S = any> {
                ~~~~~
``` 
I don't think this error is relevant to my change in this pull request.

File: libs/akita-ng-effects/src/lib/types.ts [hunk 1]
```diff
@@ -1,17 +1,22 @@
 import { Subject } from 'rxjs';
 
-
 export interface EffectOptions {
```
Comment (NetanelBasal) [inline libs/akita-ng-effects/src/lib/types.ts:3]: Thanks. Few comments:

- Set `true` (recommended) 
It's not recommended. It's based on your use-case. 

- or omit if you want the effect to dispatch an action.
If it's omitted or `false`, it'll not dispatch an action.


- Do not forget to set `false` for the effects which are not intended to dispatch.
`false` is the default.

Comment (juri-sinitson) [reply libs/akita-ng-effects/src/lib/types.ts:3]: This snippet makes `dispatch` to be `false` by default. I saw this snippet before I updated the comments.
https://github.com/datorama/akita/blob/e6f0b5d83684e9ad84ddbf065c6c571912b2c50c/libs/akita-ng-effects/src/lib/create-effect.util.ts#L9

I think you would like to see a documentation/comments as short as possible without redundant precautions. 
And of course without overseen mistakes.

```
This parameter dictates if the effect can dispatch an action. Set `true` if you want the effect to dispatch an action. Set to `false` or omit if the effect should not dispatch any action.
```

Is this formulation ok for you?

Comment (NetanelBasal) [reply libs/akita-ng-effects/src/lib/types.ts:3]: LGTM

Comment (juri-sinitson) [reply libs/akita-ng-effects/src/lib/types.ts:3]: The comment is now corrected

**Pull request merged**